src/server/http/index.js

```diff
@@ -140,6 +137,19 @@ module.exports = async function (kbnServer, server, config) {
     }
   });
 
+  server.route({
```

Comment:
Couple notes:

 - Using `server.exposeStaticDir()` will ensure that these files are served just like the rest
 - We don't control `process.env.PWD`, use `__dirname`
 - `resolve()` takes multiple arguments and combines them string interpolation is unnecessary
 - The entire bundles directory is already being served at `/bundles/`
 - Looks like the font files are being stored in the `ui/public/assets` directory, not the bundles directory

I'm thinking this sections should look something like:

```js
server.exposeStaticDir('/ui/fonts/{path*}', resolve(__dirname, '../../ui/public/assets/fonts'));
```